docs/containers-registries.conf.d.5.md

```diff
@@ -0,0 +1,33 @@
+% CONTAINERS-REGISTRIES.CONF.D(5)
+% Valentin Rothberg
+% Mar 2020
+
+# NAME
+containers-registries.conf.d - directory for drop-in registries.conf files
+
+# DESCRIPTION
+CONTAINERS-REGISTRIES.CONF.D is a systemd-wide directory for drop-in
+configuration files in the `containers-registries.conf(5)` format.
+
+By default, the directory is located at `/etc/containers/registries.conf.d`.
+
+# CONFIGURATION PRECEDENCE
+
+Once the main configuration at `/etc/containers/registries.conf` is loaded, the
+files in `/etc/containers/registries.conf.d` are loaded in alpha-numerical order.
+Specified fields in a config will overwrite any previous setting.
+
+For instance, setting the `unqualified-search-registries` in
+`/etc/containers/registries.conf.d/example.conf` will overwrite previous
```

Comment:
suggestion only, your call.  instead of `example.conf`, perhaps `myregistries.conf`?

Reply:
Sounds good!